src/components/Sidebar/FooterMenu.react.js

```diff
@@ -12,7 +12,7 @@ import React    from 'react';
 import styles   from 'components/Sidebar/Sidebar.scss';
 
 let host = location.host.split('.');
-let urlRoot = location.protocol + '//' + host.slice(host.length - 2).join('.');
+let mountPath = window.PARSE_DASHBOARD_PATH;
```

Comment:
It's set in the html header returned for all pages served by express.
[line 168 of app.js](https://github.com/JeremyPlease/parse-dashboard/blob/94b46d09a80b5c38277ce3823d03cc00cdb5258a/Parse-Dashboard/app.js#L168) and
[line 193 of app.js](https://github.com/JeremyPlease/parse-dashboard/blob/94b46d09a80b5c38277ce3823d03cc00cdb5258a/Parse-Dashboard/app.js#L193)